checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/rule/ActiveRuleWrapperImpl.java

```diff
@@ -0,0 +1,76 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2017 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 3 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package org.sonar.plugins.checkstyle.rule;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import org.apache.commons.lang.StringUtils;
+import org.sonar.api.rules.ActiveRule;
+import org.sonar.api.rules.RuleParam;
+import org.sonar.plugins.checkstyle.CheckstyleSeverityUtils;
+
+/**
+ * Wrapper as per {@link ActiveRuleWrapper} for the server side {@link ActiveRule}.
```

Comment:
`Implementation of {@link ActiveRuleWrapper} for the new server side {@link org.sonar.api.rules.ActiveRule}`

full name is used to make it more clear, as too much the same name classes.